### PR TITLE
new feature toggle for PBAv3

### PR DESCRIPTION
**Pull request by drummondjm**
Addition of new feature toggle which will be used as part of implementing PBAv3

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by filippo-possenti-version1 — DISMISSED** [review]

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by nagaraju-deekonda — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

**Review by filippo-possenti-version1 — APPROVED** [review]

**Comment by filippo-possenti-version1** [comment]
All good for me. Get a second review and it can be merged. Nothing to test here as it's functional to other features.

**Review by m-meulendijks-v1 — APPROVED** [review]

**Pull request merged**
